site/content/docs/Repositories/_index.md

```diff
@@ -52,6 +54,29 @@ gs://kopia-test-123/n417ffc2adc8dbe93f1814eda3ba8a07c
 gs://kopia-test-123/p78e034ac8b891168df97f9897d7ec316
 ```
 
+***Method #2: Creating a Service Account and Using the JSON File
+
+1. Create a storage bucket in [Google Cloud Console](https://console.cloud.google.com/storage/)
+2. Create a Google Cloud Service Account that allows you to access your storage bucket. Directions are available on [Google Cloud's website](https://cloud.google.com/docs/authentication/getting-started#create-service-account-console). Make sure to download the JSON file for your service account and keep it safe.
+
+After these preparations we can create Kopia repository (assuming bucket named `kopia-test-123`):
+
+```shell
+$ kopia repository create gcs --credentials-file="/path/to/your/credentials/file.json" --bucket kopia-test-123
+```
+
+***Method #3: Enabling AWS S3 Interoperability in Google Cloud Storage
+
+1. Create a storage bucket in [Google Cloud Console](https://console.cloud.google.com/storage/)
+2. Go to [Settings and then Interoperability](https://console.cloud.google.com/storage/settings;tab=interoperability) in your Google Cloud Storage account
+3. Enable your project under "Default project for interoperable access" and generate access keys for this project -- you will generate both access key and secret key, just like if you were using AWS S3
+
+After these preparations we can create Kopia repository using the s3 function (assuming bucket named `kopia-test-123`):
+
+```shell
+$ kopia repository create s3 --bucket="kopia-test-123" --access-key="access/key/here" --secret-access-key="secret/key/here"
```

Comment:
I think you need to pass endpoint and region here...

Reply:
You are right, sorry forgot that when copy and pasting. Updated the pull, not too familiar with github, so I hope you can see it. If not, this needs to be added to the s3 command when using s3 for gcs: `--endpoint="storage.googleapis.com"`